src/Presentation/Factory/EventFactory.php

```diff
@@ -73,8 +73,10 @@ public function createComponent(Event $event): Component
      */
     protected function getProperties(Event $event): Generator
     {
-        yield new Property('UID', new TextValue((string) $event->getUniqueIdentifier()));
+        yield new Property('UID', new TextValue((string)$event->getUniqueIdentifier()));
         yield new Property('DTSTAMP', new DateTimeValue($event->getTouchedAt()));
+        yield new Property('STATUS', new TextValue($event->getStatus()));
```

Comment:
Since `STATUS` is optional, please add a "if condition" that will check if the property needs to be added:

```suggestion
        if ($event->hasStatus()) {
            yield new Property('STATUS', new TextValue($event->getStatus()));
        }
```

The same applies to `METHOD`.

Reply:
Sure, now i fix all issues, thank you very much.

Reply:
Fixed as you suggest